tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -62,6 +62,18 @@ Login To RHODS Dashboard
    ${authorize_service_account} =  Is rhods-dashboard Service Account Authorization Required
    Run Keyword If  ${authorize_service_account}  Authorize rhods-dashboard service account
 
+Logout From RHODS Dashboard
+    [Documentation]  Logs out from the current user in the RHODS dashboard
+    ...    This will reload the page and show the `Log in with OpenShift` page
+    ...    so you want to use `Login to RHODS Dashboard` after this
+    # Another option for the logout button
+    #${user} =  Get Text  xpath:/html/body/div/div/header/div[2]/div/div[3]/div/button/span[1]
+    #Click Element  xpath://span[.="${user}"]/..
+    Click Button  xpath://div[@class="pf-c-page__header-tools"]/div/div[@class="pf-c-page__header-tools-item"][3]//button[@id="toggle-id"]
```

Comment:
Xpath looks way too big. Could we try simplifying it something like below:

`(//button[@id="toggle-id"])[1]`

Reply:
That xpath unfortunately doesn't work, it's the first one I tried

Reply:
oh, i see it would might have not worked because of indexing, below one should work

`(//button[@id="toggle-id"])[2]`